### ISSUE TITLE
Add version number to the initial log file entry

### ISSUE THREAD

**Issue by andilippi** [gottagofaster236/RewardsTheater#1]
Hey there, fantastic plugin! I just covered it on my YouTube channel and people seem to love it.

I have a plugin that checks OBS plugins for updates. I update it all manually. 

I was hoping you could in your next versions add the current version number to the obs log file on load.
This is what i use to check what version users are currently on. Then provide the download links etc.

Keep up the great work. In case you want to see what my thoughts were and additions I mention it at the end of my video:
https://youtu.be/-0evZCAlXVU

Cheers dude

**Comment by gottagofaster236** [comment]
@andilippi 
Thank you so much for a great tutorial video! Most people wouldn't care to read a long README, and my English accent wouldn't sound so nice on a video 😅 Definitely will leave a link to your video in README if you don't mind.
> I was hoping you could in your next versions add the current version number to the obs log file on load.

Would this format work for what you're trying to do?
```
[RewardsTheater] Plugin loaded, version 1.0.4
```

I'll definitely look into whether I can implement other suggestions that you and people in your comment section left 👀

**Comment by andilippi** [comment]
You are more than welcome! You are more than welcome to use the video for whatever! <3

That format is absolutely perfect. I will get it added in your next update. No rush :)

There were some great comments and ideas so I look forward to seeing what you add :)

**Comment by gottagofaster236** [comment]
Hi again,
I've added a logging on startup in the format above in 07a57b958ff3b52baf50595bdf62987025266132.
I've also added the possibility of playing random videos by using a playlist in a VLC Video Source, as you suggested in your video (thanks!)
And I've added your video to the README, so hopefully this will ease the setup for people who don't like to read a lot.
Feel free to create any new issues or DM me on Twitter if you have any new ideas!
